shared/utils/utils.js

```diff
@@ -10,3 +10,21 @@ export const throttle = (func, delay = 100) => {
     }
   }
 }
+
```

Comment:
Maybe think about having separate util files for each domain. I.e mathUtils, stringUtils, mapUtils.

It prevents module pollution